clang/docs/ClangFormatStyleOptions.rst

```diff
@@ -6992,7 +6992,7 @@ the configuration (without a prefix: ``Auto``).
     .. code-block:: c++
 
       namespace N1 {
-      namespace N2
+      namespace N2 {
```

Comment:
This file is generated from `clang/include/clang/Format/Format.h`. Please edit that file and then run `ninja clang-format-style`.